runtime/src/accounts_db.rs

```diff
@@ -5117,7 +5117,7 @@ impl AccountsDb {
                         offset,
                         ..
                     } = stored_account;
-                    if let Some((locked_entry, _)) = self.accounts_index.get(pubkey, None, None) {
```

Comment:
@carllin Assuming this is right, should we change v1 or leave it alone?

Reply:
yeah, v1 is toggled via a flag, so as long as we're supporting that flag, I think we should port these changes as well.

Reply:
Seems safe!